frontend/src/features/map_planning/layers/plant/components/PlantingElement.tsx

```diff
@@ -9,19 +10,23 @@ import {
 } from '@/features/map_planning/utils/Tooltip';
 import { isPlacementModeActive } from '@/features/map_planning/utils/planting-utils';
 import { COLOR_PRIMARY_400, COLOR_SECONDARY_400 } from '@/utils/constants';
-import { commonName } from '@/utils/plant-naming';
+import { getNameFromPlant, getPlantNameFromSeedAndPlant } from '@/utils/plant-naming';
 import { KonvaEventObject } from 'konva/lib/Node';
 import { Group, Circle, Rect } from 'react-konva';
 
 export type PlantingElementProps = {
   planting: PlantingDto;
 };
 
-const placeTooltip = (plant: PlantsSummaryDto | undefined) => {
+const placeTooltip = (plant: PlantsSummaryDto | undefined, seed: SeedDto | undefined) => {
   if (!plant) return;
 
   setTooltipPositionToMouseCursor();
-  showTooltipWithContent(commonName(plant));
+  if (!seed) {
+    showTooltipWithContent(getNameFromPlant(plant));
+  } else {
+    showTooltipWithContent(getPlantNameFromSeedAndPlant(seed, plant));
```

Comment:
I think a `getPlantName(plant, seed)` where seed is allowed to be undefined would be a bit nicer API. But this is very minor and I don't know if it would work for PlantingAttributeEditForm.tsx, so I am fine with how it is.

Reply:
If you wish, we could add this as a follow up issue to be fixed in another PR.

Reply:
No, it is very minor. It can be refactored if we would get a third argument.